### PR TITLE
Adding button xl-long class and applying it to CTA "Join Us" home page button

### LINKED ISSUE #1694: Adjust size of header CTA button on homepage
### Overview
We need to adjust the size of the large header button on the homepage because now that the text is shorter ("Join Us"), the button isn't large enough/doesn't draw enough attention compared to the button below it.

### Action Items

- [x] A new standard button class  "**.btn-xl-long**"  should be added to the design system via [_buttons.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/elements/_buttons.scss) (similar to how we have a btn-md-narrow). 
- [x] The properties for it should be:

.btn-xl-long {  
height: 72px;  
padding: 0 80px;  
border-radius: 100px;  
font-size: 1.5rem;

  @media #{$bp-below-tablet} {    
height: 60px;    
padding: 0 70px;    
border-radius: 60px;    
font-size: 1.25rem;  }}

- [x] This new standard button class should be applied to the large [homepage](https://www.hackforla.org/#) Join Us CTA button (replacing .btn-xl)

### Resources/Instructions
[Homepage](https://www.hackforla.org/)
[_buttons.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/elements/_buttons.scss)
[HfLA Site Architecture](https://github.com/hackforla/website/wiki/Hack-for-LA's-Site-Architecture)
[Standardized Components] (https://github.com/hackforla/website/wiki/Standardized-Components)

How button looks now (too small):

Screen Shot 2021-06-06 at 12 56 15 AM

How button should look:

Screen Shot 2021-06-06 at 12 56 27 AM

### PR DESCRIPTION
**Pull request by Zak234**
Fixes #1694 
Created a new button class xl-long which is the same as the xl class but with more padding. I then applied this new class to the CTA "Join US" home page button. This was done to increase the visibility of the button due to the shorter text inside.

New:
IMG-1405 (2)

Old:
IMG-1406

### CODE REVIEW

**Review by akibrhast — APPROVED** [review]
Very Nicely Done!

@Zak234   Feel free to pick up a suitable issue next!!

- Mature team members, do not merge this PR and leave this for reviewing by at least one new member
- New team members about to review , here is a guide on [How To Review A PR](https://github.com/hackforla/website/wiki/How-to-Review-Pull-Requests)

**Comment by akibrhast** [comment]
I am closing this PR temporarily for testing reasons this will be reopened soon

**Review by adam-abundis — CHANGES_REQUESTED** [review]
Please remove the closing curly brace ( } ) on line 192 and 271 within _sass/elements/_typography.scss. They not connected to any opening curly brace and they are breaking the docker build.

<details>
    <summary>Docker Build Errors</summary>
    
   <br/>
   <br/>
   
</details>

File: _sass/core/_print.scss [hunk 1]
```diff
@@ -36,6 +36,36 @@
...
+  Title5,
+  .type-Title5 {
+    font-size: #{map-get($font-size-Title5, print)};
+  }
+
+  Title6,
+  .type-Title6 {
+    font-size: #{map-get($font-size-Title6, print)};
+  }
+
```
Comment (akibrhast) [inline _sass/core/_print.scss]: @Zak234 I am not sure why these changes were made. It does not seem related to the linked issue, nor do I see any mention of this in the PR description. Could you please provide feedback in regards to this

File: _sass/elements/_typography.scss [hunk 2]
```diff
@@ -16,7 +16,15 @@ h5,
...
-ol {
+ol,
+Title1,
+Title2,
+Title3,
+Title4,
+Title5,
+Title6
+
+ {
```
Comment (akibrhast) [inline _sass/elements/_typography.scss]: @Zak234 I am not sure why these changes were made. It does not seem related to the linked issue, nor do I see any mention of this in the PR description. Could you please provide feedback in regards to this

File: _sass/elements/_typography.scss [hunk 3]
```diff
@@ -62,6 +70,42 @@ h4,
...
+  font: map-get($font-weight, bold) #{map-get($font-size-Title5, mobile)}
+  map-get($font-family, decorative);
+}
+
+Title6
+.type-Title6 {
+  font: map-get($font-weight, bold) #{map-get($font-size-Title6, mobile)}
+  map-get($font-family, decorative);
+}
+
```
Comment (akibrhast) [inline _sass/elements/_typography.scss]: @Zak234 I am not sure why these changes were made. It does not seem related to the linked issue, nor do I see any mention of this in the PR description. Could you please provide feedback in regards to this

File: _sass/elements/_typography.scss [hunk 4]
```diff
@@ -135,6 +179,36 @@ map-get($font-family, body);
...
+  Title5
+  .type-Title5 {
+    font-size: #{map-get($font-size-Title5, tablet)};
+  }
+
+  Title6
+  .type-Title6 {
+    font-size: #{map-get($font-size-Title6, tablet)};
+  }
+
```
Comment (akibrhast) [inline _sass/elements/_typography.scss]: @Zak234 I am not sure why these changes were made. It does not seem related to the linked issue, nor do I see any mention of this in the PR description. Could you please provide feedback in regards to this

File: _sass/elements/_typography.scss [hunk 5]
```diff
@@ -179,6 +253,37 @@ map-get($font-family, body);
...
+  Title5
+  .type-Title5 {
+    font-size: #{map-get($font-size-Title5, desktop)};
+  }
+
+  Title6
+  .type-Title6 {
+    font-size: #{map-get($font-size-Title6, desktop)};
+  }
+
```
Comment (akibrhast) [inline _sass/elements/_typography.scss]: @Zak234 I am not sure why these changes were made. It does not seem related to the linked issue, nor do I see any mention of this in the PR description. Could you please provide feedback in regards to this

File: _sass/variables/_typography.scss [hunk 6]
```diff
@@ -69,6 +69,48 @@ $font-size-h1: (
...
+ /* 20px*/ desktop: 1.5rem
+);
+
+$font-size-Title6: (
+ print: 12pt,
+ /* 16px*/ mobile: 1rem,
+ /* 16px*/ tablet: 1rem,
+ /* 16px*/ desktop: 1rem
+);
+
```
Comment (akibrhast) [inline _sass/variables/_typography.scss]: @Zak234 I am not sure why these changes were made. It does not seem related to the linked issue, nor do I see any mention of this in the PR description. Could you please provide feedback in regards to this

File: _sass/elements/_buttons.scss [hunk 7]
```diff
@@ -2,7 +2,7 @@
 
   Buttons
 
-  The following class names style all buttons on the HfLA site. 
+  The following class names style all buttons on the HfLA site.
```
Comment (akibrhast) [inline _sass/elements/_buttons.scss:5]: Please turn of the auto formatting feature on your work space. These line changes seems to be result of auto format

File: _sass/elements/_buttons.scss [hunk 8]
```diff
@@ -158,15 +171,14 @@
...
-
 /*----------------------------------------------------
   Page-Specific Button Classes
 
   * Use when special styling is needed for a button
   on a specific page (usually placement)
 -----------------------------------------------------*/
 
-/*   
+/*
```
Comment (akibrhast) [inline _sass/elements/_buttons.scss:181]: Please turn of the auto formatting feature on your work space. These line changes seems to be result of auto format

File: _sass/elements/_buttons.scss [hunk 9]
```diff
@@ -178,6 +190,6 @@
   btn--page-name-button-function
   EXAMPLE: btn--home-getting-started
 
-- Page-specific class names should live in that specific page's .scss file
+- Page-specific class names should live in that specific page's .scss file.
```
Comment (akibrhast) [inline _sass/elements/_buttons.scss:193]: Please turn of the auto formatting feature on your work space. These line changes seems to be result of auto format

**Review by akibrhast — APPROVED** [review]
## Requesting Changes
@Zak234 There seems to be a lot of things in this PR that is not related to this issue.
In particular, the addition of all the `title` classes. From the looks of it, I believe this was a result of you trying to implement some of the design systems from figma possibly for another [issue](https://github.com/hackforla/website/pull/1814/files), but somehow they got pushed into this issue. Could you please rectify this.. Thanks!

**Review by akibrhast — CHANGES_REQUESTED** [review]
Meant to 'request changes ' in the previous comment

**Comment by Zak234** [comment]
Yeah, sorry. Those other changes were for a different issue and I accidentally put them in the same branch. They should be removed.

**Review by akibrhast — APPROVED** [review]
LGTM!

**Pull request merged**